cli/src/lib.rs

```diff
@@ -245,6 +295,7 @@ impl Iroha {
             kura,
             block_sync,
             torii,
+            sumeragi_relay,
         })
     }
 
```

Comment:
I'm commenting in the code so that this conversation is threaded, resolved, and collapsed.

- You consider `iroha2-staging` as a staging branch for `iroha2-dev`, right?
- You could enable CI tests by, for example, renaming `iroha2-dev` in `I2::Dev::Tests` workflow
- `iroha2-staging` is outdated relative to `iroha2-dev`. When do you plan to update?

Reply:
1. Tests should be enabled in CI, because we want more than one person to contribute to this branch. 
2. We want to merge at least once every two weeks from `iroha2-dev`. Fortunately nothing concerns the new functionality, but you need to be aware which PRs might, so occasionally have a look.